scripts/repo_build.py

```diff
@@ -5,17 +5,19 @@
 
 import argparse
 import logging
+import os
+from toltec import paths
 from toltec.builder import Builder
 from toltec.repo import Repo
 from toltec.util import argparse_add_verbose, LOGGING_FORMAT
 
 parser = argparse.ArgumentParser(description=__doc__)
 
 parser.add_argument(
-    "-n",
-    "--no-fetch",
+    "-d",
+    "--diff",
     action="store_true",
-    help="do not fetch missing packages from the remote repository",
+    help="only keep new packages that do not exist on the remote repository",
```

Comment:
can you describe the difference in behavior, if any? (i think the PR artifact will still only retain the important packages - will downloads increase with this change?)

Reply:
The downloads do increase with this change. With this flag, instead of ignoring locally missing packages that already exist on the remote, the script now fetches them for the time of the build, then discards them. The rationale is that we may have some packages that depend on other existing packages at build time, so even if we do not want them in the final PR artifacts, we still need them to build the updated packages.

Do you think the flag’s description is clear enough as it stands or do you think it should be improved?

Reply:
the flag's description is fine